homeassistant/const.py

```diff
@@ -600,10 +604,21 @@ class Platform(StrEnum):
 IRRADIATION_WATTS_PER_SQUARE_METER: Final = "W/m²"
 IRRADIATION_BTUS_PER_HOUR_SQUARE_FOOT: Final = "BTU/(h×ft²)"
 
+# Precipitation intensity units
```

Comment:
Should we add a comment as to why it's different from speed?
```suggestion
# Precipitation intensity units (volume/area, over time)
```

Reply:
Yeah, we may even give the full explanation:
> The derivation of these units is a volume of rain amassing in a container with constant cross section in a given time